### PR TITLE
fix 2nd time Complete Profile cause Multiple User Uni Info Record

### LINKED ISSUE #120: Complete Profile cause Multiple User Uni Info Record
- Endpoint URI: */api/account/profile*
- Http Method: *POST*

- API Logic: 
  - When Completing User Profile should check if any User Uni Info record exist for the userId

- Progress
    - [ ] development
    - [ ] integration Test

### PR DESCRIPTION
**Pull request by kachun333**
resolve #120

### CODE REVIEW

File: src/main/java/com/thirdcc/webapp/service/impl/UserUniInfoServiceImpl.java [hunk 1]
```diff
@@ -55,12 +56,19 @@ public UserUniInfoServiceImpl(UserUniInfoRepository userUniInfoRepository, UserU
...
             .orElseThrow(() -> new BadRequestException("Cannot find user"));
+        UserUniInfo userUniInfo = userUniInfoRepository
+            .findOneByUserId(currentUser.getId())
+            .orElse(new UserUniInfo());
         userUniInfo.setUserId(currentUser.getId());
+        userUniInfo.setCourseProgramId(userUniInfoDTO.getCourseProgramId());
+        userUniInfo.setYearSession(userUniInfoDTO.getYearSession());
+        userUniInfo.setIntakeSemester(userUniInfoDTO.getIntakeSemester());
+        userUniInfo.setStayIn(userUniInfoDTO.getStayIn());
+        userUniInfo.setStatus(UserUniStatus.STUDYING);
```
Comment (HowardYaw) [inline src/main/java/com/thirdcc/webapp/service/impl/UserUniInfoServiceImpl.java]: can use or
userUniInfoDTO.getStatus() || UserUniStatus.STUDYING

**Review by HowardYaw — DISMISSED** [review]

File: src/main/java/com/thirdcc/webapp/service/impl/UserUniInfoServiceImpl.java (hunk 1 above)
Comment (kachun333) [reply src/main/java/com/thirdcc/webapp/service/impl/UserUniInfoServiceImpl.java]: this is not javascript 🤣 hahaha
but I get what you meant

Comment (kachun333) [reply src/main/java/com/thirdcc/webapp/service/impl/UserUniInfoServiceImpl.java]: fixed in f4e5ef1e934f78448db6884de1131a4a4d84589e

**Review by HowardYaw — APPROVED** [review]

File: src/main/java/com/thirdcc/webapp/service/impl/UserUniInfoServiceImpl.java (hunk 1 above)
Comment (HowardYaw) [reply src/main/java/com/thirdcc/webapp/service/impl/UserUniInfoServiceImpl.java]: I not mean the syntax
I mean the logic 😂

**Review by LUXIANZE — APPROVED** [review]

**Pull request merged**